tstest/integration/integration_test.go

```diff
@@ -300,8 +300,8 @@ func TestTwoNodes(t *testing.T) {
 	t.Logf("node1 SOCKS5 addr: %v", n1Socks)
 	t.Logf("node2 SOCKS5 addr: %v", n2Socks)
 
-	n1.AwaitListening()
 	n2.AwaitListening()
+	n1.AwaitListening()
```

Comment:
Why?

Reply:
oh, whoops. at one point I started editing the wrong test and thought I'd restored it. guess not. will revert.